File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/C/Exercicio 1.c
/* 
1 - Faça um algoritmo que receba o peso de uma pessoa, calcule e mostre:
a. O novo peso quando a pessoa engorda 15% do peso digitado
b. O novo peso quando a pessoa engorda 20% do peso digitado 
*/


#include<stdio.h>

int main ()
{
    float peso;
    printf ("Digite o seu peso: ");
    scanf ("%f",&peso);

    printf ("\nO novo peso quando engorda 15 porcento eh: %.2f",peso+(peso*0.15));
    printf ("\nO novo peso quando engorda 20 porcento eh: %.2f\n",peso+(peso*0.20));
    return 0;
}

File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/Python/exercicio1.py
""" 1 - Faça um algoritmo que receba o peso de uma pessoa, calcule e mostre:
a. O novo peso quando a pessoa engorda 15% do peso digitado
b. O novo peso quando a pessoa engorda 20% do peso digitado """

peso = float(input('Digite seu peso: '))

np15 = peso + (peso*0.15)
np20 = peso + (peso*0.20)

print('Peso mais 15% é: ' + str(np15))
print('Peso mais 15% é: ' + str(np20))